### PR TITLE
Added api button and api page

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

File: src/components/pages/api/ApiPage.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                  <th>{t('status')}</th>
+                  <th>{t('copy')}</th>
+                </tr>
+              </thead>
+              <tbody>
+                {projects.map((project) => (
+                  <tr
+                    key={project.id}
+                    className={`border-b intro-x hover:bg-gray-100 ${project.is_active ? '' : 'text-gray-500'}`}
+                  >
```
Comment (roman-tiukh) [inline src/components/pages/api/ApiPage.jsx:63]: add a transition to project view

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -240,6 +243,7 @@
   "users": "Users",
   "usersCount": "Users count",
   "view": "View",
+  "viewApiDocumentation": "View documentation",
```
Comment (roman-tiukh) [inline src/locales/en/translation.json:246]: maybe:
```suggestion
  "viewApiDocumentation": "View API documentation",
```
if key is "viewApiDocumentation"

File: src/locales/uk/translation.json [hunk 3]
```diff
@@ -240,6 +243,7 @@
   "users": "Користувачі",
   "usersCount": "Кількість користувачів",
   "view": "Переглянути",
+  "viewApiDocumentation": "Переглянути документацію",
```
Comment (roman-tiukh) [inline src/locales/uk/translation.json:246]: maybe
```suggestion
  "viewApiDocumentation": "Переглянути API-документацію",
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/components/pages/api/ApiPage.jsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  return (
+    <>
+      <div className="intro-y mt-8 p-4 pb-16 box">
+        <div className="py-4 pr-20 border-b border-gray-200">
+          <h2 className="text-lg font-medium mr-auto">API</h2>
+        </div>
+        <div className="mt-4 text-gray-700 dark:text-gray-600">
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Laudantium commodi</p>
+          <p>possimus voluptates itaque asperiores quasi, vero assumenda cupiditate</p>
+          <p>atque omnis perferendis illo expedita doloremque accusamus</p>
```
[the remark this answers is not in the document]
Comment (EmmaSamoilova) [reply src/components/pages/api/ApiPage.jsx:41]: Did like this https://www.figma.com/file/xYl9AHCL7BVesH3RJ2cU9f/API-left-menu?node-id=156%3A144

[the remark this answers is not in the document]
Comment (roman-tiukh) [reply src/components/pages/api/ApiPage.jsx:41]: I'm sure this text shouldn't be here. Its just example.
You need to know what text will be placed here

[the remark this answers is not in the document]
Comment (roman-tiukh) [reply src/components/pages/api/ApiPage.jsx:41]: https://ru.wikipedia.org/wiki/Lorem_ipsum

File: src/components/pages/api/ApiPage.jsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+            </thead>
+            <tbody>
+              {projects.map((project) => (
+                <tr
+                  key={project.id}
+                  className={`border-b intro-x hover:bg-gray-100 cursor-pointer ${project.is_active ? '' : 'text-gray-500'}`}
+                >
+                  <td onClick={() => history.push(`${projectUrl}${project.id}/`)}>{project.token}</td>
+                  <td onClick={() => history.push(`${projectUrl}${project.id}/`)}>{project.name}</td>
+                  <td onClick={() => history.push(`${projectUrl}${project.id}/`)}>{getProjectStatus(project)}</td>
```
Comment (roman-tiukh) [inline src/components/pages/api/ApiPage.jsx:67]: onClick in `tr`?

File: src/components/pages/api/ApiPage.jsx [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+  const [projects, setProjects] = useState([]);
+
+  const fetchData = () => {
+    Api.get('payment/project/')
+      .then((resp) => {
+        setProjects(resp.data);
+      });
+  };
+
+  const projectUrl = `${match.url.replace('api/', '')}profile/projects/`;
```
Comment (roman-tiukh) [inline src/components/pages/api/ApiPage.jsx:20]: You dont need this.
Just write /system/profile/projects/ in onClick function

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**